src/lib/core/fiber.h

```diff
@@ -691,6 +693,7 @@ struct fiber {
 	/* Fiber parent's backtrace allocated on the 'gc' region. */
 	struct backtrace_lua *parent_bt;
 #endif /* ENABLE_BACKTRACE */
+	bool debug_enabled;
```

Comment:
Nit: Missing a comment.

Reply:
Not applicable anymore since I dropped the field as you suggested below.